test/integration/acceptance/create_role_test.cpp

```diff
@@ -189,31 +141,30 @@ TEST_F(CreateRole, MaxLenRoleName) {
 
 /**
  * @given some user with can_create_role permission
- * @when execute tx with CreateRole command with inexistent permission name
+ * @when execute tx with CreateRole command with nonexistent permission name
  * @then the tx hasn't passed stateless validation
  *       (aka skipProposal throws)
  */
-TEST_F(CreateRole, DISABLED_InexistentPerm) {
-  IntegrationTestFramework itf;
-  itf.setInitialState(kAdminKeypair)
+TEST_F(CreateRole, DISABLED_NonexistentPerm) {
```

Comment:
I think here we have to add a note about why this test fails and add a task which fixes it.